components/invitation_modal/add_to_channels.tsx

```diff
@@ -58,6 +59,11 @@ export default function AddToChannels(props: Props) {
 
     const {formatMessage} = useIntl();
 
+    let sampleChannelName = props.currentChannel.display_name;
+    if (props.currentChannel.type !== 'O' && props.currentChannel.type !== 'P') {
```

Comment:
Can we add a comment here to provide more context on the type of channels and why we are defaulting the name.

Reply:
Hello @stevemudie, I've changed review request to you since you're more familiar with this feature. Maybe you could do a quick QA review, especially the specific change here. Thanks in advance!